src/utils/utility.ts

```diff
@@ -1,4 +1,4 @@
-import Vue from "vue";
+import * as Sentry from "@sentry/browser";
```

Comment:
Looks like an unused import

Reply:
fixed. Thanks for spotting it. I don't actually remember why it was imported in the first place.